contracts/Core/BlockManager.sol

```diff
@@ -10,10 +10,11 @@ import "./storage/BlockStorage.sol";
 import "../lib/Random.sol";
 import "../Initializable.sol";
 import "./ACL.sol";
+import "../Pause.sol";
 
 
-contract BlockManager is Initializable, ACL, BlockStorage {
-    
+contract BlockManager is Initializable, ACL, BlockStorage, Pause {
```

Comment:
Why are we inheriting Pause.sol when we are not pausing any functions in blockManager

Reply:
removed